transcript-client/src/App.tsx

```diff
@@ -1,11 +1,41 @@
 import { Box, Flex } from "@chakra-ui/react";
 import { Route, Routes, Navigate } from 'react-router-dom';
 import TranscriptionPage from "./pages/transcript";
+import { AudioProvider } from "./context/AudioContext";
 import Upload from 'src/pages/Upload';
 import Header from "./components/Header";
 import ProtectedRoute from "src/routes/protectedRoutes";
 
 function App() {
+  return (
+    <AudioProvider> {/* Wrap the entire application with AudioProvider */}
```

Comment:
Can we move the AudioProvider into the index.ts cause that is where the providers are